.github/workflows/release-custom.yml

```diff
@@ -1,16 +1,13 @@
-name: Release specific project/tag
+name: Release semanticdb
```

Comment:
why not duplicate this workflow and then customize for semanticdb? i used it to release trees specifically in the past, so it's not just for semanticdb (and, judging by the change, not at all for semanticdb).

Reply:
ok sure, I didn't realise it was useful for trees. Added a separate workflow then.